packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts

```diff
@@ -389,8 +404,11 @@ export const getVideoSenderMqa = ({videoSender, statsResults, lastMqaDataSent, b
 
   // Calculate the outgoing bitrate
   const totalBytesSentInaMin = totalBytesSent - lastBytesSent;
+  const totalRtxBytesSentInaMin = totalRtxBytesSent - lastRtxBytesSent;
 
   videoSender.common.rtpBitrate = totalBytesSentInaMin ? (totalBytesSentInaMin * 8) / 60 : 0;
+  videoSender.common.rtxPackets = totalRtxPacketsSent - lastRtxPacketsSent;
+  videoSender.common.rtxBitrate = totalRtxBytesSentInaMin ? (totalRtxBytesSentInaMin * 8) / 60 : 0;
```

Comment:
Just for the refactoring future idea. I see repetitive logic for calculating `rtxBitrate` for `videoSender` and `videoReceiver` with `* 8) / 60 : 0`. I do not know why we need to multiply by 8 first (do you know why?), and I also have an idea for the future to make this calculation a util function to avoid populating the same magic numbers across the code. I hope this idea makes sense to you

Reply:
So, that is a formula that is essentially calculating the average bitrate over a minute by converting the total bytes sent into bits and then averaging it per second. I agree with the idea that this should be refactored to have its util function. Though, I believe that this is out of the scope of this PR. We could have a Jira for this in the future.